### PR TITLE
Client now can manage realm roles

### PR DESCRIPTION
**Pull request by jakurban**
@pehala please review

### CODE REVIEW

**Comment by pehala** [comment]
I would like to see some tests for this new feature.

File: src/keycloak/admin/realmroles.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import json
+from collections import OrderedDict
+
+from keycloak.admin import KeycloakAdminBase, KeycloakAdminEntity
+
+ROLE_KWARGS = [
+    'description',
```
Comment (pehala) [inline src/keycloak/admin/realmroles.py:8]: Are these valid even for Realm roles?

File: src/keycloak/admin/realmroles.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    'composite',
+    'composites',
+    'container_id',
+    'scope_param_required'
+]
+
+__all__ = ('to_camel_case', 'RealmRole', 'RealmRoles',)
+
+
+def to_camel_case(snake_cased_str):
```
Comment (pehala) [inline src/keycloak/admin/realmroles.py]: Is this method really needed, cannot you use one in a different module?

File: src/keycloak/admin/realmroles.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    'description',
+    'id',
+    'realm_role',
+    'composite',
+    'composites',
+    'container_id',
+    'scope_param_required'
+]
+
+__all__ = ('to_camel_case', 'RealmRole', 'RealmRoles',)
```
Comment (pehala) [inline src/keycloak/admin/realmroles.py]: Is the to_camel_case method needed in __all__?

Comment (jakurban) [reply src/keycloak/admin/realmroles.py]: Yes it is used in create function

File: src/keycloak/admin/realmroles.py (hunk 1 above)
Comment (jakurban) [reply src/keycloak/admin/realmroles.py:8]: I think that they are

File: src/keycloak/admin/realmroles.py (hunk 3 above)
Comment (pehala) [reply src/keycloak/admin/realmroles.py]: but `__all__` is all the things can be improted from this module, so I do not think camel case method is neede there

**Review by pehala — APPROVED** [review]

**Pull request merged**